sql/hive/src/test/scala/org/apache/spark/sql/hive/InsertSuite.scala

```diff
@@ -225,6 +225,32 @@ class InsertSuite extends QueryTest with TestHiveSingleton with BeforeAndAfter
     checkAnswer(sql(selQuery), Row(5, 2, 3, 6))
   }
 
+  test("SPARK-37722: Escape dots in partition names") {
```

Comment:
Since it's not related to hive, shall we move the test to `org.apache.spark.sql.sources.InsertSuite`?

Reply:
I moved the test, thanks for pointing it out.